ext/afform/core/managed/AfformTags.mgd.php

```diff
@@ -26,7 +26,7 @@
         'label' => E::ts('Afforms'),
         'is_reserved' => TRUE,
         'is_active' => TRUE,
-        'domain_id' => NULL,
+        'filter' => 1,
```

Comment:
What is filter normally used for ?

Reply:
Not much of anything.

Reply:
Could it have a more semantic value? Like filter = 'tableless' and then the condition != tableless?

Reply:
Oh the table column is an `int`.. so no

Reply:
I've added a comment for the significance

Reply:
And removed the example tag